csharp/ql/test/library-tests/types/PrintAst.expected

```diff
@@ -50,29 +50,29 @@ types.cs:
 #   25|     25: [Method] DelegateType
 #   25|       -1: [TypeMention] Delegate
 #   26|     26: [Method] PointerType
-#   26|       -1: [TypeMention] Byte*
-#   26|         1: [TypeMention] Byte
+#   26|       -1: [TypeMention] byte*
+#   26|         1: [TypeMention] byte
 #   27|     27: [Method] PointerPointerType
-#   27|       -1: [TypeMention] Byte**
-#   27|         1: [TypeMention] Byte
+#   27|       -1: [TypeMention] byte**
+#   27|         1: [TypeMention] byte
 #   28|     28: [Method] PointerArrayArrayType
 #   28|       -1: [TypeMention] Byte*[][]
-#   28|         1: [TypeMention] Byte
+#   28|         1: [TypeMention] byte
 #   29|     29: [Method] NullableArrayType
-#   29|       -1: [TypeMention] Nullable
-#   29|         1: [TypeMention] Nullable<Byte>
-#   29|           1: [TypeMention] Byte
+#   29|       -1: [TypeMention] Nullable<Byte>[]
+#   29|         1: [TypeMention] byte?
+#   29|           1: [TypeMention] byte
 #   30|     30: [Method] NullableArrayArrayType
-#   30|       -1: [TypeMention] Nullable[]
-#   30|         1: [TypeMention] Nullable<Byte>
-#   30|           1: [TypeMention] Byte
+#   30|       -1: [TypeMention] Nullable<Byte>[][]
```

Comment:
`Nullable[]` vs `Nullable<Byte>[][]` is the reason why `toStringWithTypes()` was used.